src/chain.h

```diff
@@ -377,6 +389,9 @@ class CBlockIndex
         nChainTotalSupply = std::nullopt;
         nTransparentValue = std::nullopt;
         nChainTransparentValue = std::nullopt;
+        nLockboxValue = 0;
+        nChainLockboxValue = std::nullopt;
+
```

Comment:
Nit: move this after the `nChainOrchardValue` initialization so that it's in the same order as the declarations.